app/controllers/v0/health_records_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module V0
+  class HealthRecordsController < BBController
+    def refresh
+      resource = client.get_extract_status
+
+      render json: resource.data,
+             serializer: CollectionSerializer,
+             each_serializer: ExtractStatusSerializer,
+             meta: resource.metadata
+    end
+
+    def eligible_data_classes
+      resource = client.get_eligible_data_classes
+
+      render json: resource,
+             serializer: EligibleDataClassesSerializer,
+             meta: resource.metadata
+    end
+
+    def create
+      client.post_generate(params)
+
+      render nothing: true, status: :accepted
+    end
+
+    def show
+      resource = client.get_download_report('pdf')
```

Comment:
This is TBD. I can't think of a particularly good alternative to this other than the Front-end handling errors and showing a spinner despite an error having been returned, only to try the request again with a slight delay until it works. Perhaps we could batch it as a high priority sidekiq job on our side, but again I'm leaving this one as TBD for now. It's an outstanding concern of mine as well though.